Dockerfile

```diff
@@ -13,7 +13,7 @@ ENV SOURCES="file go_bindata github aws_s3 google_cloud_storage"
 
 RUN go build -a -o build/migrate.linux-386 -ldflags="-X main.Version=${VERSION}" -tags "$DATABASES $SOURCES" ./cli
 
-FROM alpine:3.8
+FROM alpine:3.12
```

Comment:
I was looking at the upstream changes and I think rather than making bespoke updates here, it would be better to pull the upstream changes. https://github.com/golang-migrate/migrate/commit/23608f1e63545d541301f9dca282bebc3a438fc7